EpiResponsivePicture/GeneratorProfiles/IResizedUrlGenerator.cs

```diff
@@ -6,5 +6,5 @@ namespace Forte.EpiResponsivePicture.GeneratorProfiles;
 
 public interface IResizedUrlGenerator
 {
-    public UrlBuilder GenerateUrl(string imageUrl, int width, PictureSource pictureSource, PictureProfile pictureProfile, FocalPoint focalPoint);
+    public UrlBuilder GenerateUrl(string imageUrl, int width, PictureSource pictureSource, ResizedImageFormat format, FocalPoint focalPoint);
```

Comment:
we should not change method signature of a public interface since this is a breaking change. Users of this package may have created their own implementations of this interface and this change will break their code.
Good news is that we don't need do change this signature as we have access to format since whole PictureProfile is passed as argument